composer.json

```diff
@@ -68,7 +68,8 @@
         "illuminate/support": "self.version",
         "illuminate/translation": "self.version",
         "illuminate/validation": "self.version",
-        "illuminate/view": "self.version"
+        "illuminate/view": "self.version",
+        "tightenco/collect": "self.version"
```

Comment:
As I understand it, `tightenco/collect` follows the version of `illuminate/support`, thus `self.version` is the correct constraint.